src/pages/reference/handlers/json-schema.md

```diff
@@ -303,13 +303,16 @@ In this example, we declare the `limit` parameter with a default value of `10` a
 -  `operations` - (required) Array of:
    -  `object`:
       -  `field` (type: `String`, required)
+         -  `operations.field` cannot contain hyphens.
       -  `description` (type: `String`)
       -  `type` (type: `String (Query | Mutation | Subscription)`, required)
       -  `requestSchema` (type: `Any`)
       -  `requestSample` (type: `Any`)
       -  `requestTypeName` (type: `String`)
       -  `responseSchema` (type: `Any`)
+         -  Remote files and URLs are not supported. You must provide a local path.
       -  `responseSample` (type: `Any`)
+         -  Remote files and URLs are not supported. You must provide a local path.
```

Comment:
Just an added note that the local path is to a file included in the `meshConfig.files` array. But I think that's understood based on basic usage of the mesh.

Also, I'm not 100% sure, but the local path might applies to more than just `responseSchema` and `responseSample. If we're ok with just mentioning it for the fields we know about then this looks good.

Reply:
I think this is good for now. If we find that more cases are impacted in the future, I'll update this section.